polar-core/src/rules.rs

```diff
@@ -94,10 +152,10 @@ impl RuleIndex {
     }
 }
 
-#[derive(Clone)]
+#[derive(Clone, Deserialize, Serialize)]
```

Comment:
Same question about serialization here.

Reply:
Thanks, good catch on these. Not needed.